input/pagecontent/mdi-content-transitions.md

```diff
@@ -0,0 +1,27 @@
+# MDI to EDRS
+
+| Name of Profile | Group | change  |  Current Profile   | New Profile  | Comment |
+| :-------------: | ----- | ------- | ------------------ | ------------ | :-----: | 
+|Cause of Death Part 1 |  MDI Profiles  |  -  | [ObservationCauseOfDeathPart1] | [ObservationCauseOfDeathPart1VitalRecordsNew] |   -   |
```

Comment:
Add Decedent, with reference to USCorePatient.   (subject field)
Add Medical Examiner/Coroner (Certifier), with reference to USCorePractitioner (attester field).